site/content/docs/docker/_index.md

```diff
@@ -17,7 +17,7 @@ links:
 
   - name: 'API Scan'
     link: api-scan/
-    desc: a full scan of an API defined using OpenAPI / Swagger
+    desc: a full scan of an API defined using OpenAPI / Swagger or GraphQL (post 2.9.0)
```

Comment:
```suggestion
    desc: a full scan of an API defined using OpenAPI / Swagger, or GraphQL (post 2.9.0)
```

Add a comma to make it just a bit more clear that the "(post 2.9.0)" only applies to GraphQL?